src/components/views/emojipicker/icons.js

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) 2016, Missive
+// From https://github.com/missive/emoji-mart/blob/master/src/svgs/index.js
+// Licensed under BSD-3-Clause: https://github.com/missive/emoji-mart/blob/master/LICENSE
+
+import React from 'react'
+
+const categories = {
+    activity: () => (
+        <svg
```

Comment:
please let's not inline the SVGs, because:
 a) consistency with the rest of the codebase
 b) caching
 c) skinning support
 d) rebranding support
 e) theming support